gwy/templates/Dockerfile.tmpl

```diff
@@ -18,7 +19,7 @@ WORKDIR /app
 # Build the gateway
 RUN go build -o grpc_gateway src/pkg/main/main.go
 
-FROM alpine:$alpine_version
+FROM alpine:$alpine
```

Comment:
Could you keep old naming(`alpine_version`)?

Reply:
I could... but all the other containers use `alpine`. I think this will be better long term if everything matches.

Reply:
I guess this is the only one that is a generated dockerfile for tho. Okay, I'll change it back

Reply:
I would say, that better to change other containers to be more explicit.